crates/meilisearch/tests/documents/add_documents.rs

```diff
@@ -2913,7 +2913,7 @@ async fn batch_several_documents_addition() {
 
     // wait first batch of documents to finish
     futures::future::join_all(waiter).await;
-    index.wait_task(4).await;
+    index.wait_task(4).await.succeeded();
```

Comment:
Here it would also be better to share the Server and get a unique index but it also means we cannot wait for the task 4 anymore.
We now have to store and retrieve the task id of the last task in the join and wait for it.